src/index.js

```diff
@@ -12,6 +12,7 @@ import renderPage from './modules/renderPage.js';
 import triggerEvent from './modules/triggerEvent.js';
 import updateTransition from './modules/updateTransition.js';
 
+import { version } from '../package.json';
```

Comment:
This throws on my (webpack) test site, where I am importing and compiling `swup/src/index.js` directly:

```shell
Can't import the named export 'version' (imported as 'version') from default-exporting module (only default export is available)
```
I have never imported a JSON file into a script, so not much experience here. This compiles for me, though:

```js
import packageJSON from '../package.json';
// ...later in the class:
version = packageJSON.version
```

Reply:
I'll have to look into this then. Bundling locally with microbundle seems fine, it should theoretically just import the actual version as a string '3.0.0'.

Reply:
Your example -- does that bundle just the version string or the whole package.json as an object? In the dist file, I mean.

Reply:
Only the ID:

```js
// CONCATENATED MODULE: ../swup/package.json
var package_namespaceObject = {"i8":"3.0.0"};
// ...
__publicField(this, "version", package_namespaceObject.i8);
```

`amdName` for example doesn't make it in the compiled file.

Reply:
I've switch to your suggested way of importing the version. Let me know if that fixes it on your end.